idea/vara-eth/frontend/src/shared/utils/styles/_breakpoints.scss

```diff
@@ -0,0 +1,38 @@
+$xl: 1200px;
+$lg: 992px;
+$md: 768px;
+$sm: 576px;
+
+@function getMaxWidth($width) {
+  @return 'max-width: #{$width - 1}';
+}
```

Comment:
No issue, just a side comment:

I'm not sure whether it's needed to subtract 1 or not - it's up to you. When I first made this decision it was based on the fact that most apps still display desktop layout at `768px` and only start to display mobile one at `767px` (probably because `min-width` or some ranges is used everywhere starting with mobile-first approach).

I would like to investigate further what's the preferred approach and would be good to know your opinion as well.